app/models/user.rb

```diff
@@ -33,11 +33,79 @@ class User < ApplicationRecord
 
   scope :remote_lawyers, -> { where(remote_clinic_lawyer: true) }
 
+  filterrific(
+    default_filter_params: { sorted_by: 'created_at_desc' },
+    available_filters: %i[
+      filter_first_name
+      filter_last_name
+      filter_email
+      with_volunteer_type
+      sorted_by
+
+    ]
+  )
+
+  def self.options_for_sorted_by
+    [
+      ['Created at (newest first)', 'created_at_desc'],
+      ['Created at (oldest first)', 'created_at_asc']
+    ]
+  end
```

Comment:
This `.options_for_sorted_by` method feels more like it should be a private method in the controller. What do you think?

Reply:
I'm not sure about this one actually. It strikes me as having a similar function as a scope, in that the action only has to do with the behavior of the data. But it really is right on the line since it's also being passed through a web request. I think I grabbed this from the filterrific documentation, so I'll double check what they did there?